lib/rubocop/cop/rspec/match_array.rb

```diff
@@ -0,0 +1,39 @@
+module RuboCop
+  module Cop
+    module RSpec
+      # Prefer `contain_exactly` when matching an array literal.
+      #
+      # @example
+      #   # bad
+      #   it { is_expected.to match_array([content1, content2]) }
+      #
+      #   # good
+      #   it { is_expected.to contain_exactly(content1, content2) }
+      #
+      #   # good
+      #   it { is_expected.to match_array([content] + array) }
+      class MatchArray < Cop
+        MSG = 'Prefer `contain_exactly` when matching an array literal.'.freeze
+
+        def on_send(node)
+          return unless node.method_name == :match_array
+          return unless node.each_child_node.all?(&:array_type?)
```

Comment:
I’m a bit confused by the `all?` here. This should only find nodes with *one* child where that child is an `array_type?`, right?